docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cisco-amp-source.md

```diff
@@ -8,7 +8,7 @@ sidebar_label: Cisco AMP
 The Cisco AMP Source provides a secure endpoint to receive data from the Cisco Amp [System Log API](https://api-docs.amp.cisco.com/api_resources?api_host=api.amp.cisco.com&api_version=v1). It securely stores the required authentication, scheduling, and state tracking information.
 
 :::note
-This Source is not available in the Fed deployment.
+This Source is available in the Fed deployment.
```

Comment:
@yleiferman , should this say FedRAMP deployment?

Reply:
Hi @kimsauce , that's a good question. The official name of the deployment is fed. FedRAMP is a certification from the federal government.

Reply:
Oh I see - thanks for clarifying! Are you ok if I hyperlink these notes with the Fed deployment docs section?